compiler/rustc_incremental/src/assert_dep_graph.rs

```diff
@@ -22,7 +22,8 @@
 //!
 //! Example:
 //!
-//! ```
+//! ```ignore HELP (can this be made to work? (well, get the desired error instead of unrelated ones I mean))
+//! # #![feature(rustc_attrs)]
```

Comment:
hmm, I'm not aware of any way to pass custom flags to the rustc invoked by rustdoc unfortunately :(